doc/api/GVT-g_api.rst

```diff
@@ -0,0 +1,114 @@
+.. _GVT-g_api:
+
+===============
+ACRN GVT-g APIs
+===============
+
+GVT-g is the GPU virtualization solution of Intel, it is open source and
+up-streamed to the Linux kernel. Its implementation over KVM is named KVMGT,
+over Xen is named XenGT and over ACRN is named ACRN-GT. GVT-g could exports
+multiple virtual GPU (alias vGPU) instances for virtual machine system (a.k.a.
+VM). A VM could be assigned one instance of vGPU. The guest OS graphic driver
```

Comment:
``of a vGPU.``

Reply:
Will change it to "on vGPU instance".   
In current code, GVT-g allows 8 vGPUs at most. And each VM has one vGPU instance.